staging/sales_index.json

```diff
@@ -1,5 +1,30 @@
 
 [
+  {
+      "chainId": 3,
+      "saleManagerAddress": "0xb2a2934de53c1dd6c330ee1db8ccb138a64837db",
+      "saleId": "0x7f1d7720c6fba223158981b82f0d9914547e2c1323a1834359015bdf2b5f7143",
+      "saleName": "Danylo (DNL)",
+      "logo": "https://tokensoft-softernet-public.s3.amazonaws.com/softernet.png",
+      "favicon": "https://tokensoft-softernet-public.s3.amazonaws.com/softernet.png",
+      "projectWebsite": "https://tokensoft.io",
+      "saleDescription": "Danylo (DNL) Public Sale",
+      "access": {
+        "limitToRegions": ["US", "GB", "CA", "UK"],
```

Comment:
Please update line 13 to read `"limitToRegions": ["US", "GB", "CA"]`
The "UK" and trailing comma should not be there (a bug in the document you copy/pasted)